aiida/workflows/arithmetic/add_multiply.py

```diff
@@ -0,0 +1,56 @@
+# -*- coding: utf-8 -*-
+###########################################################################
+# Copyright (c), The AiiDA team. All rights reserved.                     #
+# This file is part of the AiiDA code.                                    #
+#                                                                         #
+# The code is hosted on GitHub at https://github.com/aiidateam/aiida-core #
+# For further information on the license, see the LICENSE.txt file        #
+# For further information please visit http://www.aiida.net               #
+###########################################################################
+"""Basic calcfunction-based workflows for demonstration purposes."""
+
+from aiida.engine import WorkChain, calcfunction, workfunction
+
+
+@calcfunction
+def add(x, y):  # pylint: disable=invalid-name
+    return x + y
+
+
+@calcfunction
+def multiply(x, y):  # pylint: disable=invalid-name
+    return x * y
+
+
+@workfunction
+def add_and_multiply(x, y, z):  # pylint: disable=invalid-name
+    """Add two numbers and multiply it with a third."""
+    addition = add(x, y)
```

Comment:
I know `sum` is a protected keyword, but is `addition` the correct word? :sweat_smile:  It is the noun to represent the action, but not the result, right? Anyway, I am nitpicking, forget this, this is not important

Reply:
I know, it's not really the correct word. 😁But I figured it would be better to avoid using `sum`.